.github/workflows/pypi-publish.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   deploy:
-    if: github.actor == 'nficano' || github.actor == 'RONNCC' || github.actor == 'tfdahlin'
```

Comment:
we shouldn't be changing this actually -- this is written specifically so that any of the 3 of us can trigger the pypi deploy, and doesn't have to do with repo name

Reply:
erm, artifact of find/replace. good catch